wait_other.go

```diff
@@ -0,0 +1,14 @@
+// +build !linux
+
+package ev3dev
+
+import (
+	"time"
+
+	"errors"
+)
+
+// Wait is not implemented without a linux OS (needs unix.Poll)
+func Wait(d StaterDevice, mask, want, not MotorState, any bool, timeout time.Duration) (stat MotorState, ok bool, err error) {
+	return stat, false, errors.New("ev3dev: needs unix OS")
```

Comment:
Also probably worth mirroring all the actual documentation here so that godoc works - leave the line that is there at the bottom with a single blank `//` line.